app/src/main/res/layout/include_omnibar_toolbar.xml

```diff
@@ -264,277 +298,3 @@
     </com.google.android.material.appbar.AppBarLayout>
 
 </merge>
-<!--
```

Comment:
This was kept when merging develop into this feature branch to avoid extra conflicts.